cpu/esp32/Makefile.include

```diff
@@ -85,9 +59,18 @@ LINKFLAGS += -T$(RIOTCPU)/$(CPU)/ld/esp32.rom.ld
 LINKFLAGS += -T$(RIOTCPU)/$(CPU)/ld/esp32.rom.nanofmt.ld
 LINKFLAGS += -nostdlib -lgcc -Wl,-gc-sections
 
-ifneq (,$(filter esp_cxx,$(USEMODULE)))
-  UNDEF += $(BINDIR)/esp_cxx/cxa_guard.o
-endif
+# Baselibs needed when using esp_wifi module
+_ESP_WIFI_BASELIBS = -lcore -lrtc -lnet80211 -lpp -lcoexist -lwps -lwpa -lwpa2
```

Comment:
Since pseudomodule `esp_wifi_any` is enabled if either `esp_wifi`, `esp_wifi_enterprise` or `esp_now` is used, it might be better to call it `_ESP_WIFI_ANY_BASELIBS` and to define the additional `BASELIBS` for `eps_now` on top.

Reply:
So, should I change also the conditional in line 70 to check for `esp_wifi_any` instead?

Reply:
Nevermind, `esp_wifi` should be there if `esp_wifi_enterprise` is.